DuckDuckGo/NetworkProtectionTunnelController.swift

```diff
@@ -67,6 +67,10 @@ final class NetworkProtectionTunnelController: TunnelController {
         tunnelManager.connection.stopVPNTunnel()
     }
 
+    func removeVPN() async {
```

Comment:
Do we need this? What’s the difference between this and just removing it through the iOS VPN Settings?

Reply:
This exists for when the `waitlistBetaActive` feature flag is turned off, in order to clean up the VPN configuration from the system entirely. (It also removes the NetP and waitlist state at the same time.)